### PR TITLE
add modify to support user-mode Ocall

### PR DESCRIPTION
**Pull request by Shang-QY**

### CODE REVIEW

File: lib/app/include/ocall.h [hunk 1]
```diff
@@ -12,4 +12,5 @@
 #define ENCLAVE_DEFAULT_KBUFFER_SIZE              0x1000UL
 #define ENCLAVE_DEFAULT_KBUFFER         0xffffffe000000000UL
 
-#define OCALL_SYS_WRITE     3
\ No newline at end of file
+#define OCALL_SYS_WRITE     3
+#define OCALL_USER_DEFINED	4
```
Comment (Ddnirvana) [inline lib/app/include/ocall.h:16]: +@fengerhu1 Erhu, please check whether this ocall number is consistent with the TVM version.

Comment (fengerhu1) [reply lib/app/include/ocall.h:16]: In tvm, we only have a ocall that returns the relay page to the user. However, the relay page is not available in the spmp.
As ocall numbers 1-8  are already used in TVM, maybe a new ocall number 9 is fine.

Comment (Shang-QY) [reply lib/app/include/ocall.h]: Now I just modify the ocall number to 9, Thx

**Review by Ddnirvana — APPROVED** [review]
LGTM.

**Pull request merged**